### DISCUSSION TITLE
When connecting through xrdp , I am not able see the user id in output form w or who command

### DISCUSSION THREAD

**Discussion by amardeepl11** [neutrinolabs/xrdp#2876] [Q&A]
Hi All,

I am successfully connected my ubuntu Desktop on xrdp from windows system.

How ever i dot see any user details in who or w command .

I need the user id who is having access on GUI to run a zenity script to show some pop to him.

Please suggest how can i fetch this user id (who has logged in through xrdp)

**Comment by matt335672** [comment]
We've got a patch for this in #2745 which is awaiting other developments before we merge it.

Are you getting anything useful from loginctl?

**Reply by fuqifacai** [reply]
Has this feature been done?
